### PR TITLE
This commit will add b2 branch with b2NewFile1.txt

### PR DESCRIPTION
**Pull request by prasaduu**
I created a branch b2 to add a new file named 'b2NewFile1'

### CODE REVIEW

**Comment by RajaTheKing826** [comment]
endi raaa idi

**Comment by prasaduu** [comment]
> endi raaa idi

MG

**Comment by RajaTheKing826** [comment]
pedda errri p****k**** vi nuvvu

File: b2NewFile1.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Branch2
```
Comment (Nagendra7509) [inline b2NewFile1.txt:1]: Branch2 endira howle marchu firstuu..

File: b2NewFile1.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Branch2 
+New FIl1w[opjkgmrweslogjhp[oaH]]
```
Comment (Nagendra7509) [inline b2NewFile1.txt:2]: idi endira idi manishivara nuvvu neekanna arthamavuthundha idi..

Comment (prasaduu) [reply b2NewFile1.txt:2]: weight

weight ante ento telusa ra neeku

**Pull request merged**